modules/apps/web-experience/portlet-configuration/portlet-configuration-web/src/main/java/com/liferay/portlet/configuration/web/internal/portlet/PortletConfigurationPortlet.java

```diff
@@ -850,7 +850,7 @@ protected PortletPreferences getPortletPreferences(
 
 		PortletPreferencesIds portletPreferencesIds = new PortletPreferencesIds(
 			themeDisplay.getCompanyId(), layout.getGroupId(),
-			PortletKeys.PREFS_OWNER_TYPE_LAYOUT, PortletKeys.PREFS_PLID_SHARED,
+			PortletKeys.PREFS_OWNER_TYPE_GROUP, PortletKeys.PREFS_PLID_SHARED,
```

Comment:
Would it make sense to separate regular portlets from embedded portlets so that their logic don't clobber each other in the future? Even though it seems to be done by design, it's still very difficult to decipher so it might be worth changing.